app/controllers/api/v1/people_controller.rb

```diff
@@ -44,7 +44,10 @@ def court_cases
       end
 
       def timetable
-        response = People::RetrieveTimetable.call(person)
+        start_date = Date.parse(timetable_filter_params[:date_from])
```

Comment:
I'd suggest enforcing strict date validation before parsing - just in case some american software comes along and asks for `2/1/2020` (meaning 1st February, rather than 2nd January)

Reply:
🤮

Reply:
Sounds good

Reply:
Added that now :+1: